com.unity.formats.alembic/Editor/Importer/AlembicBuildPostProcess.cs

```diff
@@ -0,0 +1,109 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Security.Cryptography;
+using System.Text;
+using UnityEditor.Build;
+using UnityEditor.Build.Reporting;
+using UnityEditor.Callbacks;
+using UnityEngine.Formats.Alembic.Importer;
+using UnityEngine.SceneManagement;
+using Object = UnityEngine.Object;
+
+namespace UnityEditor.Formats.Alembic.Importer
+{
+    public static class AlembicBuildPostProcess
+    {
+        internal static readonly List<KeyValuePair<string,string>> FilesToCopy = new List<KeyValuePair<string,string>>();
+        [PostProcessBuild]
+        public static void OnPostProcessBuild(BuildTarget target, string pathToBuiltProject)
+        {
+            if (target != BuildTarget.StandaloneOSX && target != BuildTarget.StandaloneWindows64)
+            {
+                return;
+            }
+
+            foreach (var files in FilesToCopy)
+            {
+                var dir = Path.GetDirectoryName(files.Value);
+                if (dir != null && !Directory.Exists(dir))
+                {
+                    Directory.CreateDirectory(dir);
+                }
+
+                File.Copy(files.Key,files.Value, true);
+            }
+            
+            FilesToCopy.Clear();
+        }
+    }
+
+    class AlembicProcessScene : IProcessSceneWithReport
+    {
+        public int callbackOrder
+        {
+            get { return 9999;}  
```

Comment:
Is that the usual way to be sure it's the last ?

Reply:
No Other way I could find. 
The order is marginally important: if users were to instantiate more Alembics after I fixed the ones in scenes, they would simply not work in standalone builds....